src/TestFramework/AbstractTestFrameworkAdapter.php

```diff
@@ -123,34 +124,19 @@ public function getMutantCommandLine(
         );
     }
 
+    /**
+     * @throws UnsupportedTestFrameworkVersion
```

Comment:
I think it's better to throw when we cannot identify the version, rather than carrying on with the unknown version... Up for debate though

Reply:
What kind of version output do you get if you run a dev version of phpunit? I wouldn't prohibit users from using infection, just because we cannot parse the version info. We should probably output a big warning though.

Reply:
Hm fair

Reply:
Thinking about this again: it might actually be sure way better to, even in much lower versions, stick to displaying a warning about using an unsupported version rather than bailing out at all